index.js

```diff
@@ -89,3 +107,25 @@ function isEmail(email) {
     const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
     return re.test(String(email).toLowerCase())
 }
+
+function isAnonymousIdentifier(distinct_id) {
```

Comment:
another way to check is to compare `$device_id` with `distinct_id`

Reply:
@yakkomajuri: Nice! Do you want me change the code, or is it safe to merge?